functionality/database/src/main/kotlin/pl/patrykgoworowski/mintlift/functionality/database/Database.kt

```diff
@@ -0,0 +1,36 @@
+package pl.patrykgoworowski.mintlift.functionality.database
```

Comment:
```suggestion
package com.patrykandpatryk.liftapp.functionality.database
```

I’d also suggest renaming the packages accordingly.